sbt-scalajs-bundler/src/main/scala/scalajsbundler/sbtplugin/ScalaJSBundlerPlugin.scala

```diff
@@ -530,6 +556,10 @@ object ScalaJSBundlerPlugin extends AutoPlugin {
 
       npmResolutions := Map.empty,
 
+      npmExtraArgs := Seq.empty,
+
+      yarnExtraArgs := Seq.empty,
```

Comment:
I think these lines are not needed since sbt will fallback to the un-scoped settings (that you set at lines 517-518).

Reply:
Thanks I was a bit confused about having these in two places